tiledb/sm/serialization/array_schema.cc

```diff
@@ -210,21 +215,36 @@ Status attribute_from_capnp(
   RETURN_NOT_OK(
       (*attribute)->set_cell_val_num(attribute_reader.getCellValNum()));
 
+  // Set nullable.
+  const bool nullable = attribute_reader.getNullable();
```

Comment:
This needs to be wrapped in a `hasNullable`

Reply:
CapNP does not generate `hasX` for bools, maybe all primitives. We just have to be sure the default value (of `false`) works here. Leaving this as-is.

Reply:
Thanks for the update, didn't realize the `has` didn't exist for bools.